plugins/modules/source_control/github/github_repo.py

```diff
@@ -214,8 +216,8 @@ def main():
         state=dict(type='str', required=False, default="present",
                    choices=["present", "absent"]),
         organization=dict(type='str', required=False, default=None),
-        private=dict(type='bool', required=False, default=False),
-        description=dict(type='str', required=False, default=''),
+        private=dict(type='bool', required=False, default=None),
```

Comment:
It's unnecessary to state defaults for `required` & `default` :)
```suggestion
        private=dict(type='bool'),
```